infrastructure/kzg/src/main/java/tech/pegasys/teku/kzg/KZG.java

```diff
@@ -52,6 +55,8 @@ public KZGCommitment blobToKzgCommitment(final Bytes blob) throws KZGException {
 
   void loadTrustedSetup(String trustedSetup) throws KZGException;
 
+  void loadTrustedSetup(TrustedSetup trustedSetup) throws KZGException;
```

Comment:
Should this be in the interface? Could be a private method in the implementation. Or is this choice made to make testing easier?
IMO it's an implementation detail and could be hidden (And that could impact the tests: Load the test data from a file and check that this method is called with the expected parsed data)

Reply:
I wanted it to be a part of the API, but let me think again

Reply:
I'd rather move out `void loadTrustedSetup(String trustedSetup) throws KZGException;` out of the `KZG`'s API than that one. But I don't think we should over-complicate it before it's really needed, so I'd keep both for now.